core/lib/libtomcrypt/src/pk/ecc/ecc_verify_hash.c

```diff
@@ -178,13 +171,57 @@ int ecc_verify_hash(const unsigned char *sig,  unsigned long siglen,
 error:
    ltc_ecc_del_point(mG);
    ltc_ecc_del_point(mQ);
-   mp_clear_multi(r, s, v, w, u1, u2, p, e, m, NULL);
+   mp_clear_multi(v, w, u1, u2, p, e, m, NULL);
    if (mp != NULL) { 
       mp_montgomery_free(mp);
    }
    return err;
 }
 
+/**
+   Verify an ECC signature
+   @param sig         The signature to verify
+   @param siglen      The length of the signature (octets)
+   @param hash        The hash (message digest) that was signed
+   @param hashlen     The length of the hash (octets)
+   @param stat        Result of signature, 1==valid, 0==invalid
+   @param key         The corresponding public ECC key
+   @return CRYPT_OK if successful (even if the signature is not valid)
+*/
+
+int ecc_verify_hash(const unsigned char *sig,  unsigned long siglen,
+                    const unsigned char *hash, unsigned long hashlen,
+                    int *stat, ecc_key *key)
```

Comment:
I think `stat` is not needed, just return an error status instead. I.e., `CRYPT_OK` means signature could be verified and is OK, any other code means it's not or it could not be verified.
I see other functions are like this one (`dsa_verify_hash_raw()` for instance) so it could be the purpose of another patch if you think this comment is valid.